Gemfile

```diff
@@ -39,6 +39,8 @@ end
 gem 'activerecord-session_store', '~> 0.1.1'
 gem 'rails-observers', '~> 0.1'
 gem 'protected_attributes', '~> 1.1.1'
+gem 'sprockets', '~> 3'
+gem 'sprockets-rails', '~> 2.3.3', '< 3'
```

Comment:
Looks like a typo from aa50ab1, but this should be `>=` and not `~>` else it'll never get beyond 2.3.x.